app/src/main/res/values/strings.xml

```diff
@@ -189,5 +189,14 @@ We engage our community by contributing to open source, collaborating with the g
     <string name="same_request_already_sent">You have already sent a similar request to\u0020</string>
     <string name="relationpage_btntext">Find members</string>
     <string name="refresh">Refresh</string>
+
+    <string name="email_error">Email ID can\'t be left blank</string>
+    <string name="Feedback">Feedback</string>
```

Comment:
@pratikghule 
As far as I know, the string resource name should be in small letters according to the standard format.
Please take care of these so as to follow the standard form.